samples/Basic/MyGenerator/MyGenerator.csproj

```diff
@@ -7,8 +7,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-		<PackageReference Include="Uno.SourceGeneration" Version="1.28.0" PrivateAssets="All" />
-		<PackageReference Include="Uno.SourceGenerationTasks" Version="1.28.0" PrivateAssets="None" />
+		<PackageReference Include="Uno.SourceGeneration" Version="1.29.0-dev.241" PrivateAssets="All" />
```

Comment:
Note for later : As discussed with you, we should probably add the reference to the project instead of using the package reference.